msteams-platform/bots/how-to/create-a-bot-commands-menu.md

```diff
@@ -38,23 +38,23 @@ A prerequisite to create a command menu for your bot is that you must edit an ex
    > [!WARNING]
    > If you have been using App Studio, we recommend that you'd try the Developer Portal to configure, distribute, and manage your Teams apps. App Studio will be deprecated by August 01, 2022.
 
-   :::image type="content" source="conversations/Media/AppStudio.png" alt-text="appstudio-media":::
+   :::image type="content" source="conversations/Media/AppStudio.png" alt-text="The screenshot describes the steps for creating command menu using app studio.":::
```

Comment:
Let's adhere to the correct casing in all metadata/alt-text/everywhere.

Reply:
changed the casing.